util/common/tiles.py

```diff
@@ -6,10 +6,8 @@ def pos_from_name(tile, chip_size, bias):
     """
     Extract the tile position as a (row, column) tuple from its name
     """
-    size = pytrellis.IntPair()
-    size.first = chip_size[0]
-    size.second = chip_size[1]
-
+    pair = pytrellis.make_IntPair(chip_size[0],chip_size[1])
```

Comment:
why do you need to do make_IntPair and then IntPair() ? does make_IntPair not return an IntPair already?

Reply:
you are right, made multiple variations so ... :)